python/python/glide/async_commands/cluster_commands.py

```diff
@@ -909,7 +909,7 @@ async def lolwut(
         route: Optional[Route] = None,
     ) -> TClusterResponse[bytes]:
         """
-        Displays a piece of generative computer art and the Redis version.
+        Displays a piece of generative computer art and the server version.
```

Comment:
Please use valkey to be consistent, some time you use Valkey and sometime server.